src/timer/timer_enable.cc

```diff
@@ -313,6 +317,9 @@ void TimerReport::PrintToLog(unsigned indent_level) const {
     using std::chrono::duration_cast;
     using std::chrono::nanoseconds;
     const std::string indent(4 * indent_level, ' ');
```

Comment:
Better to allow a general custom prefix, not just space indentations.

Reply:
Let's do that when we have clear needs on it. It is irrelavant to this PR.